linode/instancetype/framework_models.go

```diff
@@ -137,3 +144,28 @@ func FlattenPrice(ctx context.Context, price linodego.LinodePrice) (
 
 	return &resultList, nil
 }
+
+func FlattenRegionPrices(prices []linodego.LinodeRegionPrice) (
+	*basetypes.ListValue, diag.Diagnostics,
+) {
+	result := make([]attr.Value, len(prices))
+
+	for i, price := range prices {
+		obj, d := types.ObjectValue(priceObjectType.AttrTypes, map[string]attr.Value{
+			"id":      types.StringValue(price.ID),
+			"hourly":  types.Float64Value(float64(price.Hourly)),
+			"monthly": types.Float64Value(float64(price.Monthly)),
+		})
+		if d.HasError() {
+			return nil, d
+		}
+
+		result[i] = obj
+	}
+
+	priceList, d := basetypes.NewListValue(
```

Comment:
```suggestion
	priceList, d := types.ListValue(
```

I think it might be nicer to use the higher level API wrapper.

Reply:
Oops, nice catch!